packages/react/src/FormattingToolbar/FormattingToolbarFactory.tsx

```diff
@@ -3,9 +3,24 @@ import {
   FormattingToolbarFactory,
   FormattingToolbarStaticParams,
   FormattingToolbarDynamicParams,
+  BlockNoteEditor,
 } from "@blocknote/core";
 import { FormattingToolbar as ReactFormattingToolbar } from "./components/FormattingToolbar";
 import { ReactElementFactory } from "../ElementFactory/components/ReactElementFactory";
+import { FC } from "react";
+
+export const createReactFormattingToolbarFactory = (
```

Comment:
same question as in sidemenu PR: the function call below seems very similar to what happens here